raiden/transfer/channel.py

```diff
@@ -388,6 +405,153 @@ def is_valid_lockedtransfer(
     )
 
 
+def is_valid_lockexpired(
+        state_change: ReceiveLockExpired,
+        channel_state: NettingChannelState,
+        sender_state: NettingChannelEndState,
+        receiver_state: NettingChannelEndState,
+) -> MerkletreeOrError:
+    message_name = 'LockExpired'
+    received_balance_proof = state_change.balance_proof
+    lock = channel_state.partner_state.secrethashes_to_lockedlocks.get(state_change.secrethash)
+
+    if not lock:
+        # If we don't find the lock,
+        # it must have been removed earlier.
+        return True
```

Comment:
This must return False, otherwise someone can just send an invalid remove lock message, and the balance proof will be wrongly overwritten

Reply:
Also, this needs to return a tuple of three elements, `(False, msg, None)`